### DISCUSSION TITLE
Email has already been used to obtain an academic discount on GitHub account(s)

### DISCUSSION THREAD

**Discussion by limijs** [community/community#121437] [GitHub Education]
### Hi everyone,

Prior to applying for the student developer pack on this current account, I had it on my old github account. I had 2fa enabled and I lost my authenticator app as well as the recovery codes so I unlinked both the school email and the personal email from the account. Now that I have created my new account (this is my new account) because I couldn't recover the old one, I have tried to claim the student developer benefits again going through the application and it's saying that it has already been used to obtain the student discount even though I unlinked the emails thinking I could redo the application on my new account. What should I do?

**Comment by samus-aran** [comment]
Hi @limijs 

It’s not possible to transfer Global Campus benefits over from one GitHub account to another.

However, you are able to merge your accounts, but the account with the benefits would be the main account. You can add your non-academic email from the previous account to the account containing the academic email. If you desire you can then rename this account once merged. For more details check out out the [merging of your personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) documentation.

I would suggest you resolve you old account first. Reach out to the [billing & account support team](https://support.github.com/contact/account).

**Reply by ArtemPr0** [reply]
What if I no longer have access to my old account? Is it still possible to merge or delete it? I forgot my recovery keys after having issues with 2FA so I created a new account and linked the university email to the new account (and unlinked it from old account). Still getting the error "Email has already been used to obtain an academic discount on GitHub account(s)".

Is there any way I can still get Copilot for students?

**Reply by huang1547350403** [reply]
I have the same issue, how can I apply for education account with the same email again.

**Comment by Lucas-game** [comment]
Same question!!!

**Comment by Lucas-game** [comment]
Do you have any solutions now?
